Week1/homework/databases_exercises/exercise1/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword'
+});
+
+connection.connect();
+
+connection.query('DROP DATABASE IF EXISTS `meetup`;', function (error, results, fields) {
```

Comment:
:+1: for checking for errors.

Since the callback is the same for all queries, you could've set it as a named function on the top level, and pass it in each `connection.query('...', errorCb)`.